### ISSUE TITLE
ParserCLI "-f" flag does not work as shortcut for "--from-dialect"

### ISSUE THREAD

**Issue by JanecekPetr** [jOOQ/jOOQ#9144]
### Expected behavior and actual behavior:

The `ParserCLI` does not accept any `SQLDialect`.

This sanity-test ParserCLI invokation works just fine,
```
$ java -cp "jooq-3.12.1.jar;reactive-streams-1.0.2.jar" \
        org.jooq.ParserCLI -s "select 1;"
```

but when we try to add a SQL dialect to the command, it fails.

### Steps to reproduce the problem:

This fails because the `-f` flag is defined twice in the parser, as an alias for `--formatted` and `--from-dialect`.
```
$ java -cp "jooq-3.12.1.jar;reactive-streams-1.0.2.jar" \
        org.jooq.ParserCLI -f POSTGRES -s "select 1;"

Unknown flag: POSTGRES. Use -h or --help
java.lang.RuntimeException
        at org.jooq.ParserCLI.parse(ParserCLI.java:346)
        at org.jooq.ParserCLI.main(ParserCLI.java:68)
```

I suggest renaming the `-f` (alias for `--from-dialect`) flag to `-F`. Similarly, the `-t` for `--to-dialect` should be renamed to `-T` for consistency.

---

Similarly this also fails:
```
$ java -cp "jooq-3.12.1.jar;reactive-streams-1.0.2.jar" \
        org.jooq.ParserCLI --from-dialect POSTGRES -s "select 1;"

Unknown flag: --from-dialect. Use -h or --help
java.lang.RuntimeException
        at org.jooq.ParserCLI.parse(ParserCLI.java:346)
        at org.jooq.ParserCLI.main(ParserCLI.java:68)
```

but for a different reason. This line: https://github.com/jOOQ/jOOQ/blob/master/jOOQ/src/main/java/org/jooq/ParserCLI.java#L299 contains a wrong `.toUpperCase()` call which should be two lines below:
```java
if ("-f".equals(args[i]) || "--from-dialect".equals(args[i].toUpperCase())) {
    try {
        result.fromDialect = SQLDialect.valueOf(args[++i]);
        continue argsLoop;
    }
}
```

should be
```java
if ("-f".equals(args[i]) || "--from-dialect".equals(args[i])) {
    try {
        result.fromDialect = SQLDialect.valueOf(args[++i].toUpperCase());
        continue argsLoop;
    }
}
```

### Versions:

- jOOQ: 3.12.1
- Java: 8u211
- Database: N/A
- OS: Cygwin on Windows 10
- JDBC Driver: N/A

**Comment by lukaseder** [comment]
Thanks a lot for your report. I'll split your report in two. This one here is about the `-f` flag not working. The new one (https://github.com/jOOQ/jOOQ/issues/9146) is about case insensitivity. The latter is easier to fix backwards compatibly. The former needs some additional thought. Given that `-t` has been a mandatory parameter, I'm not too keen to change it to `-T`. On the other hand, `-f` (as in `--formatted`) may have been used already, so perhaps there isn't a clean fix here from a backwards compatibility perspective.

**Comment by lukaseder** [comment]
I don't want to change `-f` (for `--formatted`). The other `-f` (for `--from-dialect`) having never worked, `-F` is fine. And we'll switch from `-t` to `-T`, while still supporting `-t` as well, without documenting it. I guess that will be good enough, given this is not *the* most popular feature.

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.13.0. Backport scheduled for 3.12.2 (#9175).

Thanks again!
